service_crategen/src/service.rs

```diff
@@ -159,10 +170,21 @@ impl<'b> Service<'b> {
                     "serde_derive".to_owned(),
                     cargo::Dependency::Simple("1.0.2".into()),
                 );
-                dependencies.insert(
-                    "serde_json".to_owned(),
-                    cargo::Dependency::Simple("1.0.1".into()),
-                );
+                // panic!("self.name is {}", self.name());
+                // some rest-json services don't use the `serde_json` crate:
+                match self.name() {
```

Comment:
Did you mean to use the `needs_serde_json_crate` function for branching here?

Reply:
I sure did! Missed this one when I pulled the function out.

Thanks!